pom.xml

```diff
@@ -186,6 +186,12 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+
+      <dependency>
+        <groupId>com.google.guava</groupId>
+        <artifactId>guava</artifactId>
+        <version>32.0.0-jre</version>
```

Comment:
by the way, current dubbo3 uses protobuf-java-util 3.25.8 which depends on guava 32.0.1-jre.